Update beta.3 Core docs - Closes #889 (#901)
LiskHQ/lisk-docs: modules/ROOT/pages/reference/cli.adoc

COMMANDS
  forging:disable  Disable forging for given delegate address.
  forging:enable   Enable forging for given delegate address.

>   forging:config   Generate the forging config object for given delegate address.

Reviewer: ```suggestion
  forging:config   Generate delegate forging config for given passphrase and password.
```

>   forging:status   Get forger info for given delegate address.

Reviewer: ```suggestion
  forging:status   Get forging information for the locally running node.
```

USAGE
  $ lisk-core forging:config

OPTIONS
  -c, --count=count            [default: 1000000] Total number of hashes to produce
  -d, --distance=distance      [default: 1000] Distance between each hashes
  -o, --output=output          Output file path

  -p, --passphrase=passphrase  Specifies a source for your secret passphrase. Command will prompt you for input if this option is not set.
                               	Examples:
                               	- --passphrase='my secret passphrase' (should only be used where security is not important)

  -w, --password=password      Specifies a source for your secret password. Command will prompt you for input if this option is not set.
                               	Examples:
                               	- --password=pass:password123 (should only be used where security is not important)

  --pretty                     Prints JSON in pretty format rather than condensed.

EXAMPLES
  forging:config
  forging:config --password your_password
  forging:config --passphrase your_passphrase --password your_password --pretty
  forging:config --count=1000000 --distance=2000 --output /tmp/forging_config.json
----

> 

Reviewer: The forging:status command seems to be missing?

Author: it's coming in the next step
